app/components/c-prices.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div :class="b()">
+    <dl :class="b('price-definition')">
+      <dt :class="b('price-label')">{{ $t('c-prices.grossLabel') }}</dt>
+      <dd :class="b('price-value')">{{ displayPriceGross }}</dd>
+    </dl>
+    <dl :class="b('price-definition')">
+      <dt :class="b('price-label')">{{ $t('c-prices.netLabel') }}</dt>
+      <dd :class="b('price-value')">{{ displayNetPrice }}</dd>
+    </dl>
+  </div>
+</template>
+
+<script>
+
+  /**
+   *
+   * This component does not fetch any data from ERP, it just displays
+   * the prices provided to it as attributes and formats them according to the
+   * currently used language.
+   * This means that the parent component is responsible for fetching the data from ERP.
+   * If there should be no prices shown, just hide the whole component by using v-if.
+   *
+   */
+  export default {
+    name: 'c-prices',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * The gross price as delivered by ERP
+       */
+      priceGross: {
+        type: Number,
+        required: true,
+      },
+
+      /**
+       * The net price as delivered by ERP.
+       */
+      price: {
+        type: Number,
+        required: true,
+      },
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      displayPriceGross() {
+        return this.$n(this.$props.priceGross / 100, 'currency', 'de-DE'); // TODO - remove hardcoded locale
```

Comment:
How you handle the local in genereal? I would recommend to add a function to the i18n mixin which get's you the current locale